tests/endpoint.py

```diff
@@ -130,6 +138,11 @@ def stats_correct_data(set_data):
     set_data('Short_DATA_STATS_PATH')
     return survival
 
+@pytest.fixture()
+def external_correct_data(set_data):
+    set_data('SHORT_DATA_EXTERNAL_PATH')
```

Comment:
@microbe73 @grugna The readme should be updated explaining the new variables.

Reply:
You are right, thank you @paulmurdoch19.
@microbe73  can you push those changes.